src/lib/users/ca_password_validator.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2016 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require "yast"
+
+module Users
+  # Validator to check if a password fulfills the requirements to
+  # generate CAs (see FATE#300438)
+  class CAPasswordValidator
+    MIN_LENGTH = 4
+    private_constant :MIN_LENGTH
+
+    Yast.import "ProductFeatures"
```

Comment:
is there reason to have import inside class? it is a bit unintuitive for me as I expect it in require place and usually place it elsewhere only when there is benefit from it ( like rarely usage or circular dependencies )